src/Sylius/Bundle/ShopBundle/Resources/config/routing/product.yml

```diff
@@ -13,3 +13,16 @@ sylius_shop_product_show:
                 arguments:
                     - $slug
                     - 'expr:service("sylius.context.channel").getChannel()'
+
+sylius_shop_product_reviews_index:
+    path: /products/{slug}/reviews
+    methods: [GET]
+    defaults:
+        _controller: sylius.controller.product_review:indexAction
+        _sylius:
+            template: SyliusShopBundle:Product/Review:index.html.twig
+            repository:
+                method: findAcceptedByProductSlug
+                arguments:
+                    - $slug
+                    - '%locale%'
```

Comment:
All shops routes should take into account channel, right now I can display reviews for products that are disabled for my channel if only I know the `slug`